includes/Widget.php

```diff
@@ -40,7 +40,7 @@ public function widget( $args, $instance ) {
 		// Prepare options.
 		$title     = ! empty( $instance['title'] ) ? $instance['title'] : __( 'Editors', 'wp-team-list' );
 		$role      = ! empty( $instance['role'] ) ? $instance['role'] : 'editor';
-		$show_link = isset( $instance['show_link'] ) ? $instance['show_link'] : false;
+		$show_link = $instance['show_link'] ?? false;
```

Comment:
The operator is only available in PHP 7.0+ but the minimum is defined as 5.6.  Either bump the required PHP version or revert this change.

Reply:
Ok, I will wait till PHPCS 4.0 is released and change the config `<config name="testVersion" value="5.6-"/>`